exports.js

```diff
@@ -460,6 +460,7 @@ module.exports = {
         'advancedDataSecurityEnabled'   : require(__dirname + '/plugins/azure/sqlserver/advancedDataSecurityEnabled.js'),
         'tdeProtectorEncrypted'         : require(__dirname + '/plugins/azure/sqlserver/tdeProtectorEncrypted.js'),
         'noPublicAccess'                : require(__dirname + '/plugins/azure/sqlserver/noPublicAccess.js'),
+        'privateEndpointsConfigured'    : require(__dirname + '/plugins/azure/sqlserver/privateEndpointsConfigured.js'), 
```

Comment:
```suggestion
        'serverPrivateEndpoints'        : require(__dirname + '/plugins/azure/sqlserver/serverPrivateEndpoints.js'), 
```

Reply:
I like this name better because this is consistent with naming convention. You need to change the name of files as well